src/Footer.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+
+const Footer = props => {
+  return (
+    <div className="footer">
+      <ul>
+        {props.info.map(function(item, index) {
```

Comment:
Try to stick to one way of declaring functions, e.g. arrow functions (usually the cleanest way inside JSX)